### PR TITLE
fix: fp8 config

### PR DESCRIPTION
**Pull request by Ying1123**

### CODE REVIEW

File: python/sglang/srt/managers/controller/model_runner.py [hunk 1]
```diff
@@ -118,6 +120,11 @@ def load_model(self):
             seed=42,
             skip_tokenizer_init=True,
         )
+        print(vllm_model_config.hf_config.num_key_value_heads)
+        if vllm_model_config.model == "meta-llama/Meta-Llama-3.1-405B-FP8":
```
Comment (zhyncs) [inline python/sglang/srt/managers/controller/model_runner.py]: Should we consider the local path?

File: python/sglang/srt/managers/controller/model_runner.py [hunk 2]
```diff
@@ -118,6 +120,11 @@ def load_model(self):
             seed=42,
             skip_tokenizer_init=True,
         )
+        print(vllm_model_config.hf_config.num_key_value_heads)
```
Comment (zhyncs) [inline python/sglang/srt/managers/controller/model_runner.py]: nit: We may delete it.

File: python/sglang/srt/managers/controller/model_runner.py [hunk 3]
```diff
@@ -118,6 +120,11 @@ def load_model(self):
...
+        if vllm_model_config.model == "meta-llama/Meta-Llama-3.1-405B-FP8":
+            vllm_model_config.hf_config.num_key_value_heads = 8
+        print(vllm_model_config.hf_config.num_key_value_heads)
```
Comment (zhyncs) [inline python/sglang/srt/managers/controller/model_runner.py]: nit: We may delete it.

File: python/sglang/srt/managers/controller/model_runner.py [hunk 4]
```diff
@@ -118,6 +120,11 @@ def load_model(self):
...
+        exit()
```
Comment (zhyncs) [inline python/sglang/srt/managers/controller/model_runner.py]: Is it for debug?

Comment (Ying1123) [reply python/sglang/srt/managers/controller/model_runner.py]: yes. it is still in progress.

Comment (zhyncs) [reply python/sglang/srt/managers/controller/model_runner.py]: ok

**Pull request merged**